rebasehelper/build_tools/koji_tool.py

```diff
@@ -79,7 +81,11 @@ def creates_tasks(cls):
 
     @classmethod
     def _scratch_build(cls, source, **kwargs):
-        session = cls.koji_helper.session_maker()
+        try:
+            session = cls.koji_helper.session_maker()
+        except requests.exceptions.SSLError as e:
+            raise RebaseHelperError('{}\n{}'.format(six.text_type(e),
+                                                    'You probably do not own a correct active kerberos ticket.'))
```

Comment:
This is not right. What actually happens is that [`koji_session.krb_login()`](https://github.com/rebase-helper/rebase-helper/blob/51755dd20b8240f7abe28676953d38315795eb51/rebasehelper/utils.py#L903) raises `koji.krbV.Krb5Error` and then falls back to authenticate using [`koji_session.ssl_login()`](https://github.com/rebase-helper/rebase-helper/blob/51755dd20b8240f7abe28676953d38315795eb51/rebasehelper/utils.py#L906), but that is long unsupported and you probably can't even get valid certificate to authenticate through it now.
It might be best to drop the fallback entirely and instead raise `RebaseHelperError` with an explanation.

Reply:
Also, there is currently plan to deprecate `python-krbV` in Fedora, which will probably affect `koji` and the `koji.krbV.Krb5Error` exception:
https://fedoraproject.org/wiki/Changes/kerberos-in-python-modernization

Reply:
Alright, so I guess raising `RebaseHelperError` with a message that koji session was not created possibly because of missing kerberos ticket, will be better.

Reply:
There should be some error message associated with `koji.krbV.Krb5Error`, so I would just use that.